operator/cmd/mesh/manifest-apply.go

```diff
@@ -144,8 +146,12 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, maArgs *manifestApplyAr
 	if err := configLogs(logOpts); err != nil {
 		return fmt.Errorf("could not configure logs: %s", err)
 	}
-	if err := ApplyManifests(applyFlagAliases(maArgs.set, maArgs.manifestsPath, maArgs.revision), maArgs.inFilenames, maArgs.force, rootArgs.dryRun,
-		maArgs.kubeConfigPath, maArgs.context, maArgs.readinessTimeout, l); err != nil {
+	client, err := kube.NewClient(kube.BuildClientCmd(maArgs.kubeConfigPath, maArgs.context))
+	if err != nil {
```

Comment:
`K8sConfig` creates a client with increased QPS/Burst to reduce install time. This behavior should probably be preserved.

Reply:
+1, LGTM otherwise

Reply:
Added the configuration to kube client creation.